package.json

```diff
@@ -34,6 +33,7 @@
     "koa-route": "2.4.2",
     "koa-send": "3.1.0",
     "lodash": "3.10.1",
+    "mongodb": "^2.1.4",
```

Comment:
Remove the caret `^` to use exact version in line with the rest of the dependencies. We update deps manually after a review and testing not to break existing stuff.